rbac/management/migrations/0001_initial.py

```diff
@@ -16,24 +15,54 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name="Access",
             fields=[
-                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
```

Comment:
please don't change init migration even it is only indentation - if you want to do it I recommend to do it in other PR